### DISCUSSION TITLE
Save as PDF - real 1:1 content, not like "print to pdf"

### DISCUSSION THREAD

**Discussion by mahcin** [Floorp-Projects/Floorp#723] [Feature Request]
Hello,
first of all thank you for creating Floorp, it's brilliant web browser :-D
I just switched from Opera. I had Firefox years ago, but this Floorp is the best :-D
I'm wondering if would be possible to create "Save as PDF" feature under mouse right button click to save content as a pdf, but real one, not "print" one? The difference is fact that "print" or any possible extension will not save www 1:1. There always are some small or big differences. I don't know how the manage it, but in Opera this works perfect, always 1:1 save. I think it makes copy on www somewhere based on server as I don't need to scroll down the page to make everything loaded, it will still save whole content, 
after few secs it shows in downloads folder rather than ordinary action while doing any form of "print to pdf". It does make clickable links in pdf file, it's 100% 1:1
This little feature I'm using so often, it would be brilliant to have it in Floorp. None Firefox extension is able to do it as good as Opera browser, so I'm not expecting it and using double browsers at the moment, however it's worth to ask and mention about :-)

I'm attaching simply print screen of exampled web page + pdf file done with Opera as "Save to PDF" which you can compare between views.

[BBC - Home.pdf](https://github.com/Floorp-Projects/Floorp/files/13974786/BBC.-.Home.pdf)
![bbc](https://github.com/Floorp-Projects/Floorp/assets/155994915/4d6ae3d8-092b-4ac0-9b3a-d7eb3a71a87a)

All the best
/Martin

**Comment by Telmesomething** [comment]
There are a few addons like that on the Moz site: https://addons.mozilla.org/en-US/firefox/search/?q=Save%20as%20PDF
Take your pick.

**Reply by mahcin** [reply]
Thanks for suggestion but before creating this query I have checked every single addon, honestly. It took me about 2h and unfortunately none of them can do it 1:1 like Opera does, that's why I'm so much wondering what's so unique stuff Opera use to make this so perfect, including clickable links in saved pdf.

Obviously my main browser is Floorp and will be Floorp. I'm using backup browser (Opera) for saving info (www) as pdf, it would be nice to have it in Floorp so then I could stop using Opera. No worries if that's too low priority, worth to ask :-)
